README.md

```diff
@@ -136,8 +136,8 @@ second transaction that indeed the balance is 42 (0x2a).
 # Architecture
 
 The compiler uses `rustc`, `llvm`,
-[eth-riscv-syscalls](https://github.com/leonardoalt/r55/tree/main/eth-riscv-syscalls),
-[eth-riscv-runtime](https://github.com/leonardoalt/r55/tree/main/eth-riscv-runtime)
+[eth-riscv-syscalls](https://github.com/r55-eth/r55/tree/main/eth-riscv-syscalls),
+[eth-riscv-runtime](https://github.com/r55-eth/r55/tree/main/eth-riscv-runtime)
 and [riscv-rt](https://github.com/rust-embedded/riscv/tree/master/riscv-rt) to
 compile and link ELF binaries with low-level syscalls to be executed by
 [rvemu-r55](https://github.com/lvella/rvemu):
```

Comment:
We should also update this one to r55 org

Reply:
Thanks for letting me know!